src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -41,23 +66,54 @@ describe('Alerts (Unit)', () => {
     await app.close();
   });
 
+  it('returns 200 (OK) for valid signature/valid payload', async () => {
+    const alert = alertBuilder().build();
+    const timestamp = Date.now().toString();
+    const signature = fakeTenderlySignature({
+      signingKey,
+      alert,
+      timestamp,
+    });
+
+    await request(app.getHttpServer())
+      .post('/alerts')
+      .set('x-tenderly-signature', signature)
+      .set('date', timestamp)
+      .send(alert)
+      .expect(200);
```

Comment:
We should probably assert the body too even if empty. That way if we start returning something this test would fail.

Reply:
I've added the expectation in 40b5b7e3bc64689c32778bf570fe515d9ba972a7.